job-roles/solution-engineer.md

```diff
@@ -0,0 +1,29 @@
+Solution Engineer
+==================
+
+At MayaData we believe that the best way to deliver storage and
+related services to containerized and cloud-native environments is
+with containerized and cloud-native architectures. Everything we do is
+focused on providing data agility and simplifying the daily operation
+of Kubernetes SRE’s and DevOps teams using Kubernetes as a data
+layer. We enable our users to handle complex data management
+operations easily, and we have become open source leaders in building
+Kubernetes into a first-class data platform.
+
+The Role
```

Comment:
```suggestion
### Requirements
- Share our [values](https://github.com/mayadata-io/culture/blob/master/plow.md), and work in accordance with those values
  * Positive and solution-oriented mindset
  * Effective communication skills: Regularly achieve consensus with peers, and clear status updates
  * An inclination towards communication, inclusion, and visibility
  * Self-motivated and self-managing, with strong organizational skills.
  * Demonstrated ability to work closely with other parts of the organization
  * Demonstrated ability to onboard and integrate with an organization long-term
  * Ability to thrive in a remote-first organization
  * Comfortable working in a highly agile, intensely iterative software development process
- Computer science education or equivalent experience.
- Each Solution Engineer will be expected to be an expert with one or more platforms and workloads based on the professional experience and role.
- Professional experience with any other technologies that may be required by the expertise or specialty.
- Proficient in the microservices architecture, ability to debug microservices, ability to explain and understand concepts around Kubernetes, DevOps and Data Management. 
- Proficiency in the English language, both written and verbal, sufficient for success in a remote and largely asynchronous work environment
- Demonstrated capacity to clearly and concisely communicate about complex technical, architectural, and/or organizational problems and propose thorough iterative solutions
- Experience with usability, performance and optimization problems and a demonstrated ability to both diagnose and prevent these problems
- Experience owning a project from concept to production, including proposal, discussion, and execution. Ability to work single-handed as well as in a team.
- Ability to use MayaData Products - OpenEBS, Litmus and Kubera
 
### Nice-to-haves
- Experience in a peak performance organization, preferably a tech startup. 
- Experience with the MayaData products - OpenEBS, Litmus, Kubera as a user or contributor
- Experience working with a global, remote team or otherwise multicultural team
- Passionate about working for Infrastructure Products focused on Storage and/or Kubernetes SREs
- Passionate about/experienced with open source and developer tools


### Responsibilities
```